src/clockcache.c

```diff
@@ -863,16 +863,41 @@ clockcache_inc_ref(clockcache *cc, uint32 entry_number, threadid counter_no)
    debug_assert(refcount != MAX_READ_REFCOUNT);
 }
 
+/*
+ * Caller-macro to figure out who's calling this dec_ref interface.
+ *
+ * clockcache_dec_ref(clockcache *cc, uint32 entry_number, threadid counter_no)
+ */
+#define clockcache_dec_ref(cc, entry_number, counter_no)                       \
```

Comment:
This change isn't directly related to this fix, per se, but if you invoke:

`driver_test splinter_test --perf --num-insert-threads 63 --num-lookup-threads 63 ...`

We will run into an assertion on L893, complaining about ref-count going to 0 after a dec-ref.

I need to log a diff issue to trace this, and so this instrumentation is added to help debug that separate issue.

Reply:
I don't see the assertion you're talking about on L893 either in the before or after of this PR.

Reply:
Reverted the debug code as suggested. New diffs to be uploaded soon.

I don't recall which assertion was tripping. I should have documented it better here when I ran into it.
I'll try to repro this later on ...